<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QWidget" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>561</width>
        <height>316</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::FieldsStayAtSizeHint</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="formAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="terminalLabel">
         <property name="text">
          <string>Terminal</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="terminalEnableAfterInstall">
         <property name="text">
          <string>Show After Install:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="terminalEnableAfterInstallCheckBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>On</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="terminalDeviceLabel">
         <property name="text">
          <string>Device:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="terminalDeviceComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="terminalDeviceLockCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="terminalDeviceRefreshButton">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="terminalDevicePinConfigLabel">
         <property name="text">
          <string>Pin Assign:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QSpinBox" name="terminalDevicePinConfigSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="uartStopLabel">
         <property name="text">
          <string>Stop Bits:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QComboBox" name="uartStopComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1.5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="uartParitylabel">
         <property name="text">
          <string>Parity:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QComboBox" name="uartParityComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>none</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>odd</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>even</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="uartBaudrateLabel">
         <property name="text">
          <string>Baud rate:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QComboBox" name="uartBaudrateComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>57600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>76800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>230400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>460800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>921600</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="terminalLogFileLabel">
         <property name="text">
          <string>Log File:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="CLineEdit" name="terminalLogFile" native="true">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CLineEdit</class>
   <extends>QWidget</extends>
   <header>CSdk/CLineEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
